bambi/models.py

```diff
@@ -572,7 +583,10 @@ def plot_priors(
         axes: matplotlib axes or bokeh figures
         """
         if not self.built:
```

Comment:
I think we do not need this anymore as a model object is instantiated with both data and formula and thus sampling from the priors and priors predictive distributions should be immediately available.

Reply:
I thought the same when applying the changes. But if you do something like 
```python
model = Model("y ~ x", data)
model.set_priors(common=Prior("Normal", sd=10))
```
You need to call `model.build()` before that prior affects the backend. 

Another option is to call `.build()` in `.set_priors()` but what if we set priors several times? That's why the message says `"Call .build() to build the model or .fit() to build and sample from the posterior."`